src/back/gql/models/users/types.js

```diff
@@ -16,6 +16,9 @@ export const UserType = new GraphQLObjectType({
     email: {
       type: GraphQLString,
     },
+    username: {
```

Comment:
Similar to **email**, shouldn't this be unique?

Reply:
Read above ^^